packages/components/atoms/f-button/test-utils/component-objects/f-button.component.js

```diff
@@ -5,30 +5,21 @@ module.exports = class Button extends Page {
         super('atom-folder', 'f-button--button-component');
     }
 
-    get actionComponent () { return $('[data-test-id="action-button-component"]'); }
-
-    open (url) {
-        super.open(url);
-    }
+    get component () { return $('[data-test-id="action-button-component"]'); }
```

Comment:
I feel like this file should be renamed to `f-button--action.component.js` to match the convention of the other component objects.